flink-connectors/flink-sql-connector-hive-3.1.2/src/main/resources/META-INF/NOTICE

```diff
@@ -8,3 +8,51 @@ This project bundles the following dependencies under the Apache Software Licens
 
 - org.apache.hive:hive-exec:3.1.2
 - org.apache.thrift:libfb303:0.9.3
+
+The bundled Apache Hive org.apache.hive:hive-exec dependency bundles the following dependencies under
+the Apache Software License 2.0 (http://www.apache.org/licenses/LICENSE-2.0.txt)
+
+- com.esotericsoftware:kryo-shaded:3.0.3
+- com.esotericsoftware:minlog:1.3.0
+- com.esotericsoftware:reflectasm:1.10.1
+- com.google.guava:guava:19.0
+- com.googlecode.javaewah:JavaEWAH:0.3.2
+- com.tdunning:json:1.8
+- commons-lang:commons-lang:2.6
+- io.airlift:aircompressor:0.10
+- javax.jdo:jdo-api:3.0.1
+- joda-time:joda-time:2.9.9
+- net.sf.opencsv:opencsv:2.3
+- org.apache.avro:avro-mapred:1.8.2
+- org.apache.avro:avro:1.8.2
+- org.apache.commons:commons-lang3:3.3.2
+- org.apache.hive.shims:hive-shims-0.23:3.1.2
+- org.apache.hive.shims:hive-shims-common:3.1.2
+- org.apache.hive:hive-common:3.1.2
+- org.apache.hive:hive-llap-client:3.1.2
+- org.apache.hive:hive-llap-common:3.1.2
+- org.apache.hive:hive-metastore:3.1.2
+- org.apache.hive:hive-serde:3.1.2
+- org.apache.hive:hive-service-rpc:3.1.2
+- org.apache.hive:hive-spark-client:3.1.2
+- org.apache.hive:hive-standalone-metastore:3.1.2
+- org.apache.hive:hive-storage-api:3.1.2
```

Comment:
I don't know what's going on with my maven

```
[INFO] Including org.apache.hive:hive-storage-api:jar:2.7.0 in the shaded jar.
```

Reply:
You are right, I made mistake.